lib/pbench/test/unit/server/database/test_audit_db.py

```diff
@@ -31,11 +31,10 @@ def fake_db(self, monkeypatch, server_config):
         server configuration object directly on the Database.Base (normally
         done during DB initialization) because that can't be monkeypatched.
         """
-        self.session = FakeSession(Audit)
-        with monkeypatch.context() as m:
-            m.setattr(Database, "db_session", self.session)
-            Database.Base.config = server_config
-            yield
+        __class__.session = FakeSession(Audit)
+        monkeypatch.setattr(Database, "db_session", self.session)
```

Comment:
You replaced `self` with `__class__` at line 34 but not at line 35.  We should be consistent, here.

And, if you remove both `self` references, then you should probably change this to be either a `@classmethod` (in which case you can use `cls` instead of `__class__` 😉) or a `@staticmethod` (oooh, stacked decorators!...what order should they go in??  😉)